### ISSUE TITLE
addon-a11y - make "VIOLATIONS" and "PASSES" meet AA or AAA levels of accessibility 

### ISSUE THREAD

**Issue by jchiem** [storybooks/storybook#4835]
Not sure if this is an actual bug, so I will file this as a Feature request since there's no actual problem with the functionality of the addon in this issue. 

**Problem:** 
@storybook/addon-a11y is shows if a story is accessible, but the addon itself is inaccessible for the red and green text for the aXe results tabs. It is not reaching WCAG 2.1 AA contrast ratio requirements for these two elements.

**Solution:**
Change the green `#0edf62` to `#09833a`
Change the red `#ff3f3f` to `#d53535` (

Or some other accessible colours, the colours I am suggesting are bare minimums, alternatively change the background to make the current red and green more readable.

This should let the text used in the addon reach at least WCAG 2.1 AA contrast ratio requirements and in my opinion makes sense since it's an accessibility addon.

**Comment by shilman** [comment]
@jchiem this makes sense, tho I don't know much about a11y. I put together a quick PR to change the success/fail colors in the default theme
